pulsar-broker/pom.xml

```diff
@@ -432,9 +432,15 @@
           <failOnViolations>true</failOnViolations>
           <javaVersion>17</javaVersion>
           <ignorePackages>
-            <ignorePackage>org.apache.pulsar.broker</ignorePackage>
+            <ignorePackage>org.apache.pulsar.broker.admin</ignorePackage>
+            <ignorePackage>org.apache.pulsar.broker.namespace</ignorePackage>
+            <ignorePackage>org.apache.pulsar.broker.service</ignorePackage>
+            <ignorePackage>org.apache.pulsar.broker.stats</ignorePackage>
             <ignorePackage>org.apache.pulsar.client</ignorePackage>
           </ignorePackages>
+          <exclusionPatterns>
+            <exclusionPattern>java/lang/StringBuffer."&lt;init&gt;":.*</exclusionPattern>
```

Comment:
@youzipi may I ask the reason you add this pattern?

Reply:
there is a conflict between mockito and moderinizer:
moderinizer prefers `StringBuilder` instead of `StringBuffer` **always**,
but `Mockito.doReturn(new StringBuilder("http://127.0.0.1:8080")).when(request).getRequestURL();` will fail in CI as type not match.

former discussion:
https://github.com/apache/pulsar/pull/17275#discussion_r963809645